app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -323,12 +341,32 @@ const SurveyObservations = () => {
                 Resolve the following errors in your local file and re-import.
               </Typography>
             </Box>
+
+            <Box>
+              {Object.entries(submissionErrors).map(([key, value], index) => {
```

Comment:
can just do:
```suggestion
              {Object.entries(submissionErrors).map(([key, value], index) => (
```
and then remove the `return` statement